src/static/riot/search.tag

```diff
@@ -696,26 +689,29 @@
         <div class="description">
             <p>{description}</p>
         </div>
-        <div class="extra" style="margin-top: 0;">
-            <span style="font-size: .8em; color: rgba(0,0,255, 0.6);">{url}</span>
-            <span style="font-size: .8em;">
+        <div class="extra">
+            <div class="url_date">
+            <span id="item_url">{url}</span>
```

Comment:
Eep, my comment may have been lost? Sorry if reposting this:

We should probably use ["semantically named classes."](https://css-tricks.com/semantic-class-names/) Like, `class="url"` or just making it an `h1` and styling that within `competition-tile`.

Also, since these appear many times on the page we shouldn't be using `id=`